runelite-client/src/main/java/net/runelite/client/plugins/menuentryswapper/MenuEntrySwapperPlugin.java

```diff
@@ -456,7 +461,7 @@ else if (config.swapBoxTrap() && option.equals("take"))
 		{
 			swap("lay", option, target, true);
 		}
-		else if (config.swapChase() && option.equals("pick-up"))
+				else if (config.swapChase() && option.equals("pick-up"))
```

Comment:
Fix the indentation and squash to 1 commit.

Reply:
Wow yeah I'm not sure why it does that, It's not like that in my IDE. About to get dinner but will be on it in a few.